### DISCUSSION TITLE
unable to use without odoo demo data

### DISCUSSION THREAD

**Discussion by chesty** [Tecnativa/doodba-copier-template#512] [Q&A] [answered]
in a nutshell, how to create an odoo without demo data?

long story
when I follow https://github.com/Tecnativa/doodba-copier-template/blob/main/docs/daily-usage.md and run

`invoke develop img-build git-aggregate resetdb start`

everything works great, but my new odoo has demo data. 

I found in devel.yaml

```
      # To avoid installing demo data export DOODBA_WITHOUT_DEMO=all
      WITHOUT_DEMO: "${DOODBA_WITHOUT_DEMO-false}"

```

when I set DOODBA_WITHOUT_DEMO=all then rerun the invoke command, I can see WITHOUT_DEMO=all is set in the environment of the running odoo container, but there's still demo data. 

As far as I can tell (which isn't very far) `invoke resetdb` uses  click-odoo-initdb like

```
            f"{_run} click-odoo-initdb -n {dbname} -m {modules}",
```

and that defaults to demo data, there's an optional switch to disable it

**Comment by pedrobaeza** [comment, marked as answer]
Run with `DOODBA_WITHOUT_DEMO=all` in your environment `docker compose run --rm odoo -i base` instead of `resetdb`.

**Comment by chesty** [comment]
cheers. i thought I was going crazy. that exact command didn't work but I was able to the equivalent a different way. thank you.
